### ISSUE TITLE
Support for running tests in series

### ISSUE THREAD

**Issue by mikeblum** [loadimpact/k6#723]
In an effort to modularize and re-use code / create atomic testing situations I've broken out our performance tests into separate files:

```
├── config.json
├── company
│   ├── auth.js
│   ├── context.js
│   ├── company.js
│   ├── process.js
│   └── utils.js
├── run_tests.sh
└── scenarios
    ├── test_dashboards.js
    ├── test_login_client_credentials.js
    ├── test_login_password.js
    ├── test_processes.js
    ├── test_roles.js
    └── test_users.js
```

and the shellscript's job is to declare any `ENVIRONMENT_VARIABLES` and run the JS test ex.

```
k6 run \
    -e TEST_USERNAME=user@example.com \
    -c config.json  \
    scenarios/test_dashboards.js
```

It would be a useful feature to be able to run these tests in series by passing in the root tdir that holds all of the tests. Something like:

`k6 run --test-dir=scenarios`

Some potential hang-ups would be the metrics and deeply nested test directories.

**Comment by na--** [comment]
Something like this would be possible after https://github.com/loadimpact/k6/pull/1007 is done and merged, by just having different executors for each scenario. Controlling whether the scenario execution would be in series or in parallel can be done via the `startTime` option for each executor.

Not sure if I should leave this issue open after the PR is merged though, because while running multiple load testing scenarios in a single k6 run will be possible, there probably will be some room for improvement of the UX of using that feature...

**Comment by na--** [comment]
So, #1007 was [released in k6 v0.27.0](https://github.com/loadimpact/k6/blob/master/release%20notes/v0.27.0.md#new-execution-engine-1007), and it indeed introduced [`scenarios`](https://k6.io/docs/using-k6/scenarios). They are not perfect, but the missing features to have proper "test suites" (as well as the obstacles to implement them) are covered in https://github.com/loadimpact/k6/issues/1342, so please follow that issue for updates.
